docs/02-get-started/01-quick-install.md

```diff
@@ -4,7 +4,20 @@ title: Quick install
 
 To get started with Kyma, let's quickly install it first.
 
-## Install Kyma
+Kyma project is currently in the transition phase from classic to modular Kyma. You can either install classic Kyma with its components, or available modules. To see a list of Kyma modules, go to [Overview](/docs/01-overview/README.md).
+
+> **CAUTION:** Components transformed into modules aren't installed as part of the preconfigured classic Kyma.
+
+## Install Kyma modules
```

Comment:
```suggestion
## Install a Kyma module
```

So it matches the first sentence.

Btw, can users install several modules at once? Just for my curiosity.
If they can, we can keep the original headline too.

Reply:
Absolutely, Nina, good point, thanks